sdcm/utils/cql_utils.py

```diff
@@ -0,0 +1,26 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2024 ScyllaDB
+
+import re
+
+
+def cql_quote(identifier: str) -> str:
+    """
+    quote cql identifier as needed
```

Comment:
actually I need to change it a bit, it's not just start with digit
it's more complicated then that, it should quote also if some parts are uppercase

since CQL is lowercaseing anything automatic if it's not quoted

Reply:
let's put it to docstring - will be easier than trying decipher regex. 
and name still is not relevant.

Reply:
update docstring, and changed the name